sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntityRelationType.cs

```diff
@@ -13,68 +13,68 @@ namespace Azure.AI.TextAnalytics
     public class HealthcareEntityRelationType : IEquatable<HealthcareEntityRelationType>
     {
         /// <summary>
-        /// Specifies that the entity corresponds to a Person.
+        /// Specifies the relation type of DirectionOfBodyStructure.
         /// </summary>
         public static readonly HealthcareEntityRelationType DirectionOfBodyStructure = new HealthcareEntityRelationType("DirectionOfBodyStructure");
 
         /// <summary>
-        /// Specifies that the entity corresponds to a job type or role held by a person.
+        /// Specifies the relation type of DirectionOfExamination.
```

Comment:
I think it will read better without the `of`. so `Specifies the relation type DirectionOfExamination.`
Same in other places